curite_form/src/main.rs

```diff
@@ -0,0 +1,32 @@
+use clap::Parser;
+use std::fs;
+use warp::Filter;
+use handlebars::Handlebars;
+use serde_json::json;
+
+#[derive(Parser)]
+#[clap(author, version, about, long_about = None)]
+struct Opts {
+    port: u16,
+    path: String,
+}
+
+#[tokio::main]
+async fn main() {
+    let args = Opts::parse();
+
+    let mut hb = Handlebars::new();
+
+    hb.register_template_string(
+        "page",
+        fs::read_to_string(args.path).expect("couldn't read template file"),
+    )
+    .expect("couldn't register template");
+
+    let page = warp::path!(String / String).map(move |account, token| {
+        hb.render("page", &json!({"account": account, "token": token}))
+            .expect("couldn't format template")
```

Comment:
Should this maybe make it redirect to the failure page?

Reply:
im fairly certain the only things in this file that can fail is reading the template on start up and binding to the port on start up, so i'm not sure it is worth making this script aware of where the failure page is

Reply:
I'm not sure what the failure modes for `.render()` are (presumably depends on how filling the template goes at runtime), but does a panic from the expect here kill the server? If so, it might be better to err on the side of caution and include one, so that if there is a mistake in the template that triggers it that goes un-noticed, it gives users a message to let us know.

If not, the first time it's triggered will down the server and people won't get a message indicating to let us know there's a problem.